### DISCUSSION TITLE
Website not reflecting changes

### DISCUSSION THREAD

**Discussion by LinaAoyama** [daattali/beautiful-jekyll#922] [Q&A]
I've completed initial setup (fork & rename the repo), then committed changes to _config.yml, index.html, and aboutme.md, then waited 30 minutes, but the changes are still not reflected on my website.

I've checked the Actions tab and confirmed that the CI job appears to have run successfully:
https://i.imgur.com/DVLkeX6.png

How else can I troubleshoot?

**Comment by daattali** [comment]
I do see your title as "Lina's website"
It takes a few minutes
